tests/test-ssh-remote-command.cmd

```diff
@@ -1,4 +1,4 @@
-footloose config create --config %t.footloose --name %t --key %t-key
-footloose create --config %t.footloose
-%out footloose --config %t.footloose ssh node0 hostname
-footloose delete --config %t.footloose
+../footloose config create --config %t.footloose --name %t --key %t-key
+../footloose create --config %t.footloose
+%out ../footloose --config %t.footloose ssh root@node0 whoami
```

Comment:
Yes, that tests seems great to me to sure we do login as root. You've added it as a separate test, which is nice, but we shouldn't then change the `test-ssh-remote-command` test.

`test-ssh-remote-command.*` should be left untouched.